.travis.yml

```diff
@@ -1,5 +1,6 @@
+dist: trusty
 language: python
-sudo: false
+sudo: required
```

Comment:
There's a [sudo-less beta](https://docs.travis-ci.com/user/trusty-ci-environment/#Using-Trusty) for trusty.. did you try this?